package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "sgmap.ludwig",
+  "name": "sgmap-ludwig",
```

Comment:
I had a lot of trouble to use that package and I was worried that the name was part of the issue. I also thought that the `.` was not standard (cf Mes Aides packages).

Your suggestion is much better, I will move toward it.